Src/FluentAssertions/Xml/XDocumentAssertions.cs

```diff
@@ -237,6 +265,81 @@ public AndConstraint<XDocumentAssertions> NotBeEquivalentTo(XDocument unexpected
             return new AndWhichConstraint<XDocumentAssertions, XElement>(this, xElement);
         }
 
+        /// <summary>
+        /// Special overload for <see cref="HaveElement(string, OccurrenceConstraint, string, object[])"/> with
```

Comment:
🔧 The fact that this overload uses `Exactly.Once()` is totally irrelevant to a consumer. Please focus on the functional purpose of this method.